libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/index.ts

```diff
@@ -92,6 +93,9 @@ class TigerWorkspaceElementsQuery implements IElementsQuery {
                 const response = await this.authCall((client) => {
                     const elementsRequest: ElementsRequest = {
                         label: ref.identifier,
+                        filterBy: {
```

Comment:
Should this always be there or only if any filtering is happening?

Reply:
filterBy is now apply only when filtering by uris